extension/src/test/ExternalResources.test.ts

```diff
@@ -164,4 +160,25 @@ suite("External Resources Tests", function () {
       `File "${langCode.pristine}.json" should exist`
     );
   });
+
+  test("Blob storage authentication failed", async function () {
+    const expiredToken =
+      "sv=2019-12-12&ss=f&srt=o&sp=r&se=2021-11-25T05:28:10Z&st=2020-11-24T21:28:10Z&spr=https&sig=JP3RwQVCZBo16vJCznojVIMvPOHgnDuH937ppzPmEqQ%3D";
+    const blobContainer = new BlobContainer(exportPath, baseUrl, expiredToken);
+    blobContainer.addBlob("sv-se");
+
+    await assert.rejects(
+      async () => {
+        await blobContainer.getBlobs();
+      },
+      (err) => {
+        assert.strictEqual(err.name, "Error");
+        assert.strictEqual(
+          err.message,
+          "Blob storage authentication failed. Please report this as an issue on GitHub (https://github.com/jwikman/nab-al-tools)."
+        );
+        return true;
+      }
+    );
+  });
```

Comment:
I think I've finally gotten a hang of the error assertions. It was a bit of a struggle but once I realized that the whole chain must be asynchronous it fell into place :+1:

Reply:
Nice!
Then you might find out what needs to be done to fix below error as well? It's been in the pipeline for some time...
![image](https://user-images.githubusercontent.com/23032645/143676126-ffcfcde9-fe91-43e2-8ba4-796b5f3fc141.png)

(and if you are looking through the logs, there are other errors as well that I don't know why they're in there 🙂)

Reply:
I was actually hunting for that one yesterday. I think I'm getting closer.

Reply:
It's like chasing ghosts, it seems... 😂